imagepipeline-base/src/main/java/com/facebook/imageformat/ImageFormatCheckerUtils.java

```diff
@@ -59,5 +59,31 @@ public static boolean startsWithPattern(
     return true;
   }
 
+  /**
+   * Checks if byteArray interpreted as sequence of bytes contains the pattern.
+   * @param byteArray the byte array to be checked
+   * @param pattern the pattern to check
+   * @return index of beginning of pattern, if found; otherwise -1
+   */
+  public static int containsPattern(
```

Comment:
We can just use `String.contains()` here. No need for writing an O(n^2) find substring.